src/components/canrawsender/canrawsender_p.h

```diff
@@ -1,70 +1,48 @@
 #ifndef CANRAWSENDER_P_H
 #define CANRAWSENDER_P_H
 
+#include "newlinemanager.h"
 #include "ui_canrawsender.h"
-#include <memory>
-#include <QtSerialBus/QCanBusFrame>
 #include <QtGui/QStandardItemModel>
+#include <memory>
 
 namespace Ui {
 class CanRawSenderPrivate;
 }
 
 class CanRawSenderPrivate : public QWidget {
-    Q_OBJECT
-    Q_DECLARE_PUBLIC(CanRawSender)
+  Q_OBJECT
+  Q_DECLARE_PUBLIC(CanRawSender)
 
 public:
-    CanRawSenderPrivate(CanRawSender* q)
-    : ui(std::make_unique<Ui::CanRawSenderPrivate>())
-    , q_ptr(q)
-    {
-        ui->setupUi(this);
+  CanRawSenderPrivate(CanRawSender *q)
+      : ui(std::make_unique<Ui::CanRawSenderPrivate>()), q_ptr(q) {
+    ui->setupUi(this);
 
-        tvModel.setHorizontalHeaderLabels({"id", "value",""});
-        ui->tv->setModel(&tvModel);
+    tvModel.setHorizontalHeaderLabels({"Id", "Data", "Cyclic", "Loop", ""});
```

Comment:
1. Move "Loop" before "Cyclic"
2. Change Cyclic to Interval
3. When Loop is not checked Interval should be disabled i.e. "greyed out"